Fix linter warnings

When you run the linter, you still see these warnings:

```s
$ npm run lint

> lint
> eslint --ext ".ts,.js,.vue" --ignore-path .gitignore .


/home/josecelano/Documents/git/committer/me/github/torrust/torrust-index-frontend/components/Markdown.vue
  2:22  warning  'v-html' directive can lead to XSS attack  vue/no-v-html

/home/josecelano/Documents/git/committer/me/github/torrust/torrust-index-frontend/components/form/FormInputText.vue
  12:21  warning  The "update:modelValue" event has been triggered but not declared on `defineEmits`  vue/require-explicit-emits

✖ 2 problems (0 errors, 2 warnings)
```

- [x] Fix linter warning:  'v-html' directive can lead to XSS attack
- [x] Fix linter warning:  The "update:modelValue" event has been triggered but not declared on `defineEmits`
- [ ] Add a workflow to run linter on push/pr events

--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -1,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+    target-branch: 'develop'
+
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: daily
+    target-branch: 'develop'

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -1,0 +1,26 @@
+name: Test
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Lint
+        run: npm run lint
+
+      - name: Build
+        run: npm run build
+
+      - name: Generate
+        run: npm run generate
+
+

--- a/README.md
+++ b/README.md
@@ -1,5 +1,7 @@
 # Torrust Index Frontend
 
+[![Test](https://github.com/torrust/torrust-index-frontend/actions/workflows/test.yml/badge.svg)](https://github.com/torrust/torrust-index-frontend/actions/workflows/test.yml)
+
 This repository serves as the frontend for the [Torrust Index](https://github.com/torrust/torrust-index) project.
 
 It contains the [vue-3](https://vuejs.org/) frontend web application for the [Torrust Index](https://github.com/torrust/torrust-index).

--- a/components/Markdown.vue
+++ b/components/Markdown.vue
@@ -1,4 +1,5 @@
 <template>
+  <!-- eslint-disable vue/no-v-html -->
   <div class="prose" v-html="sanitizedDescription" />
 </template>
 

--- a/components/form/FormInputText.vue
+++ b/components/form/FormInputText.vue
@@ -1,13 +1,13 @@
 <template>
   <div class="flex flex-col items-center">
-    <label class="label w-full">
-      <span class="label-text self-start">{{ props.label }}</span>
+    <label class="w-full label">
+      <span class="self-start label-text">{{ props.label }}</span>
     </label>
     <input
       :type="props.type"
       :placeholder="props.placeholder"
       :value="modelValue"
-      class="input input-bordered border-2 rounded-2xl w-full"
+      class="w-full border-2 input input-bordered rounded-2xl"
       :required="props.required"
       @input="$emit('update:modelValue', $event.target.value)"
     >
@@ -15,6 +15,10 @@
 </template>
 
 <script setup lang="ts">
+import { defineProps, defineEmits } from "vue";
+
+const emit = defineEmits(["update:modelValue"]);
+
 const props = defineProps({
   type: {
     type: String,

--- a/pages/torrents.vue
+++ b/pages/torrents.vue
@@ -60,10 +60,9 @@
 </template>
 
 <script setup lang="ts">
-import { useRuntimeConfig } from "nuxt/app";
+import { useRuntimeConfig, useRoute, useRouter } from "nuxt/app";
 import { TorrentCompact } from "torrust-index-types-lib";
 import { Ref } from "vue";
-import { useRoute, useRouter } from "#app";
 import { computed, onMounted, ref, useTags, watch } from "#imports";
 import { useCategories, useRestApi } from "~/composables/states";
 import { TorrustSelectOption } from "~/components/TorrustSelect.vue";

--- a/project-words.txt
+++ b/project-words.txt
@@ -1,1 +1,2 @@
 Nuxt
+proxied

